provisioning/roles/php-apache/tasks/php7.yml

```
@@ -6,12 +6,11 @@
   notify: restart apache
   become: yes
 
-#- name: Configure mod-php
-#  command: php5enmod {{ item }}
-#  with_items:
-#    - php-dev
-#    - xdebug
-#  notify: restart apache
-#  become: yes
+- name: Configure mod-php
+  command: "{{ phpenmod }} {{ item }}"
```

```suggestion
  command: "{{ phpenmod }} php-dev"
```
And remove the `with_items` below.